docs/pages/lookback_zscore.md

```diff
@@ -0,0 +1,94 @@
+
+## Choosing the historic distribution of results
+
+When scoring a contender benchmark result, in order to control for variables that don't have to do with actual performance changes, each result in the historic distribution must share the contender's:
+
+- repository
+- hardware
+- case permutation
+- context
+
+Results with errors will not be included, even if they also have data.
+
+Additionally, when using the lookback z-score method, one must provide a baseline commit (or baseline result or baseline run, which is ultimately associated with a baseline commit).
+The historic distribution will be drawn from all runs on commits in the baseline commit's git ancestry, up to and including all runs on the baseline commit itself.
+The maximum number of commits used is defined by the Conbench server's `DISTRIBUTION_COMMITS` parameter, which defaults to 100.
+In certain cases, there will be fewer than this number: if there are not that many commits in the ancestry, or if a distribution change was annotated by someone acting as a data curator -- see below for more details.
+
+In some cases (e.g. navigating the UI, or generating alerts with the `benchalerts` library), Conbench will provide the user with a sensible baseline commit.
+
+- If the contender is on a PR branch, the sensible baseline commit is the point at which the PR branch forked from the default branch.
+- If the contender is on the default branch, the sensible baseline commit is the parent commit.
+- If the contender is not associated with the git graph for some reason, the sensible baseline commit is the latest commit on the default branch that has benchmark results.
+
+If there are no matching results on the baseline commit, that's alright - the analysis will still include matching results from the git history.
+If there are multiple results on the baseline commit (from different runs, etc.), all of them will be used.
+
+## Creating the z-score
+
+To create a z-score, Conbench must calculate the mean and standard deviation of the historic result distribution.
+Before doing so, outliers are automatically excluded from the distribution.
+Documentation on this process is coming soon.
+
+After outliers are excluded, the remaining data is sorted by commit timestamp.
+For each benchmark result, Conbench uses the mean of all the result's iterations.
+Then:
+
+- the rolling mean is calculated (with the window boundaries as described above)
+- the residual of each result's mean from the rolling mean is calculated
+- the standard deviation of the residuals is calculated, using Bessel's correction
+
+Then the z-score is the difference of the distribution's mean from the contender result's mean, divided by the standard deviation of the residuals.
+The sign is modified depending on the unit, such that a positive z-score is "good" and a negative z-score is "bad."
```

Comment:
Could we replace good and bad with improvement and regression here?

Reply:
Done!